set.mm

```diff
@@ -37287,6 +37287,15 @@ technically classes despite morally (and provably) being sets, like ` 1 `
   elun2 $p |- ( A e. B -> A e. ( C u. B ) ) $=
     ( cun ssun2 sseli ) BCBDABCEF $.
 
+  $( A statement is true for every element of the union of a pair of classes if
+     and only if it is true for every element of the first class and for every
+     element of the second class.  (Contributed by BTernaryTau,
+     27-Sep-2023.) $)
+  elunt $p |- ( ( C e. ( A u. B ) -> ph ) <->
```

Comment:
Is there a list of conventions anywhere? I've been trying to follow them when I recognize them but I often don't (e.g. I had no idea that `t` was used for the closed theorem form).

Reply:
Usually, the conventions are documented in Section "Conventions" in set.mm.

The usage of the suffix "t", however, is not documented there. It's explained/discussed in the Google group: https://groups.google.com/g/metamath/c/4TSHENMYqtw/m/0ikUglHxAAAJ. If I understand it correctly, it was once a convention, but should generally not be used anymore (ony if the name without suffix is used already).

Reply:
Huh, I did find https://us.metamath.org/mpeuni/conventions.html, but I completely missed the link to the label conventions page even though it's right there at the top.

Reply:
"antecedent" was sometimes translated into labels using `*ant`, e.g., https://us.metamath.org/mpeuni/embantd.html or `*s`, e.g. https://us.metamath.org/mpeuni/alcoms.html. The suffix `*ant` is certainly more descriptive, but a bit long. The suffix `*t`, however, is used for closed forms ("theorems"), and the suffix `*a` for conjunctions, e.g., https://us.metamath.org/mpeuni/jca.html.
No strong opinions; I'd leave @avekens see what is best.

Reply:
To be honest, I do not understand why this should be an "antecedent theorem", and why the suffix "ant" for "antecedent" should be used. It's a biconditional with two implications by an element of a union, so maybe ~ elunimb would be a better name (and is as long as ~ elunant). @benjub would you agree?

Reply:
Furthermore, the suffix "ant" is not used in Main (only in mathboxes), therefore it seems not to be a general convention to use it.

Reply:
I just linked to https://us.metamath.org/mpeuni/embantd.html above, which is not in a mathbox. This is about "antecedent" since this is about applying elun to the antecedents of each side of the biconditional. The current "elunant" seems good, actually. But your proposal is fine too. As you prefer.

Reply:
> I just linked to https://us.metamath.org/mpeuni/embantd.html above, which is not in a mathbox. 

OK, but this is the only usage in Main, and the label corresponds 1-to-1 to the description "Deduction embedding an antecedent."

>This is about "antecedent" since this is about applying elun to the antecedents of each side of the biconditional. 

Yes, in this way it is indirectly about "antecedents", but in my opinion not enough to mention it in the label.

> The current "elunant" seems good, actually. But your proposal is fine too. As you prefer.

I would like to hear a third opinion...

Reply:
Maybe @tirix's ?

Reply:
I'd say this theorem is about **el**ements of a **un**ion acting as an **ant**ecedent, so `elunant` sound good.

But as is well known, _there are only two hard things in Computer Science: cache invalidation and naming things_.